Remove deprecated Starter and Team tiers from docs (#7930)
FlowFuse/flowfuse: docs/cloud/billing.md

## Billing Cycle

> Teams are billed monthly on the anniversary of the team creation. You will receive one bill for each team.

Author: Billing Cycle / Removing / Suspended sections: dropped the deprecated plan names ('Starter plan teams', 'For Team plan teams') across the four sentences here and below. Billing mechanics are unchanged.

Reviewer: ```suggestion
If you've signed up to FlowFuse Cloud without a yearly contract agreed upon with our sales team, your FlowFuse teams are billed monthly on the anniversary of the team creation. You will receive one bill for each team.
```

> Node-RED Instances and Devices are added as pro-rated charges on the current billing cycle and invoiced

Reviewer: ```suggestion
Hosted and Remote Instances which are added beyond the included amount are pro-rated charges on the current billing cycle and invoiced
```

at the end of the cycle.

## Removing Instances

> When a Node-RED instance is deleted your account will receive pro-rated credit for the time remaining in the billing cycle.

Reviewer: ```suggestion
When an instance is deleted your account will receive pro-rated credit for the time remaining in the billing cycle.
```
